index.html

```diff
@@ -24,30 +30,21 @@
 
       TODO: purchase font license(s) and integrate directly via @font-face
     -->
-    <link
-      rel="stylesheet"
-      href="https://use.typekit.net/avt3hiq.css"
-    />
+    <link rel="stylesheet" href="https://use.typekit.net/avt3hiq.css" />
 
     <!-- Geo Blocking script from geotargetly.com -->
     <script>
-      (function(g,e,o,t,a,r,ge,tl,y){
-        t=g.getElementsByTagName(o)[0];y=g.createElement(e);y.async=true;
-        y.src='https://g9904216750.co/gb?id=-NkqVZbqVT7_Wp1sgHo5&refurl='+g.referrer+'&winurl='+encodeURIComponent(window.location);
-        t.parentNode.insertBefore(y,t);
-      })(document,'script','head');
-    </script>
-
-    <!-- Hotjar Tracking Code for https://explorer.aptoslabs.com/ -->
-    <script>
-      (function(h,o,t,j,a,r){
-          h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
-          h._hjSettings={hjid:3271013,hjsv:6};
-          a=o.getElementsByTagName('head')[0];
-          r=o.createElement('script');r.async=1;
-          r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
-          a.appendChild(r);
-      })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
+      (function (g, e, o, t, a, r, ge, tl, y) {
+        t = g.getElementsByTagName(o)[0];
+        y = g.createElement(e);
+        y.async = true;
+        y.src =
+          "https://g9904216750.co/gb?id=-NkqVZbqVT7_Wp1sgHo5&refurl=" +
```

Comment:
Seems like this is still using the same old key? Would this be an issue?

Reply:
Good catch.  I went ahead and removed it.  we dont need to do any geotargeting right now.